ratpack-pac4j/src/test/groovy/ratpack/pac4j/openid/OpenIdRpSpec.groovy

```diff
@@ -46,6 +48,17 @@ class OpenIdRpSpec extends RatpackGroovyDslSpec {
     }
 
     handlers {
+      prefix 'prefix', {
+        configurePac4j delegate
+      }
+
+      configurePac4j delegate
+    }
+
+  }
+
+  GroovyChain configurePac4j(GroovyChain chain) {
```

Comment:
@erdi I've updated the spec to use GroovyChainAction and made the main test a bit more readable.